src/lib.rs

```diff
@@ -1558,4 +1563,7 @@ pub struct RegallocOptions {
 
     /// Run the SSA validator before allocating registers.
     pub validate_ssa: bool,
+
+    /// Run the SSRA algorithm
+    pub use_fastalloc: bool,
```

Comment:
Rather than a `bool`, can we define an `enum Algorithm { Ion, Fastalloc }` and take an option `pub algorithm: Algorithm`? This leaves room for adding others in the future and makes the intent more clear.

(We can keep the `Default` derivation working by adding `#[default]` to the `Ion` arm of the enum)